LEGO1/lego/legoomni/src/race/legoracespecial.cpp

```diff
@@ -12,6 +12,18 @@
 
 // File name verified by BETA10 0x100cedf7
 
+
+// FUNCTION: LEGO1 0x10012de0
+void LegoCarRaceActor::FUN_10012de0()
+{
+	// This function is probably in the wrong place.
+	// According to the address it should be in legoracespecial.h.
+	// However, if we put it there, it gets inlined instead of called.
```

Comment:
This is my biggest question. Any idea on how to get this function to be linked somewhere within `legoracespecial.h` without it getting inlined into calling functions?

Reply:
Maybe I'm missing something, but according to the address this function should be located between `LegoRaceCar::Notify` and `LegoRaceCar::SetWorldSpeed`, no? I know this would mean the function belongs to `LegoRaceCar` instead of `LegoCarRaceActor` but that makes sense.

The 3 static variables that are set inside the function should then also be moved to `legoracers.cpp`

Reply:
Yes, you are absolutely right. I somehow missed that this function is indeed located in the middle of a cpp file, I only saw the rather close functions in header files. Should be fixed now.